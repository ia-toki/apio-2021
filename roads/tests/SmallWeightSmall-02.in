c10234cabdfd6eae47773fb6f163e0350fc797e1
200
184 117 8
129 184 1
96 99 9
154 182 9
27 186 4
53 110 8
23 0 8
195 130 10
143 140 2
196 163 4
186 89 5
2 118 2
79 47 10
164 6 6
168 114 7
59 6 10
106 99 4
60 27 5
192 125 1
38 149 2
123 26 8
116 16 9
166 34 9
51 45 10
25 7 2
14 158 6
119 190 1
92 151 9
11 193 7
30 99 9
146 90 8
140 173 5
87 94 1
91 152 5
135 171 2
100 77 5
46 186 2
177 40 5
13 152 4
187 144 1
49 173 4
5 129 7
194 143 10
69 144 1
36 61 7
8 143 6
136 143 2
52 57 6
26 67 7
56 144 3
185 166 7
15 40 5
139 6 8
148 162 5
121 120 7
111 56 5
28 96 8
158 43 2
115 40 5
7 194 10
22 122 5
120 153 4
114 194 4
181 128 8
88 158 10
55 164 3
35 112 4
21 90 10
191 52 6
1 72 3
16 15 5
169 56 5
31 60 10
85 48 10
63 22 8
152 95 5
48 187 5
112 79 7
37 99 1
167 48 8
198 140 10
173 189 1
89 190 1
4 158 9
165 45 1
145 94 4
156 22 1
124 129 1
78 144 6
160 53 5
151 46 10
138 24 3
153 184 1
65 125 8
45 69 1
104 84 6
81 18 2
137 154 2
74 20 4
147 174 2
83 122 10
107 83 4
24 90 3
157 162 3
84 59 2
108 27 8
113 53 8
32 19 5
68 160 3
67 75 5
134 103 4
12 184 5
50 171 8
82 25 8
150 114 7
42 73 6
99 105 3
128 186 8
62 25 2
29 73 6
72 187 7
197 87 6
66 156 2
190 129 2
102 99 4
43 30 1
188 63 1
149 175 5
39 83 10
90 109 8
9 19 6
47 94 7
144 82 2
20 43 10
76 14 7
110 135 3
54 189 7
33 99 9
58 150 5
41 69 3
71 113 2
159 116 7
122 69 5
118 115 7
94 178 8
189 164 4
95 89 6
3 52 2
172 0 7
64 72 5
163 8 8
132 59 1
103 154 2
44 47 10
142 49 7
193 112 1
80 92 5
176 107 7
171 38 4
182 120 9
77 141 8
86 196 1
70 109 6
75 62 7
61 152 3
178 53 8
126 82 9
133 22 9
130 196 5
105 56 5
0 81 3
183 140 1
73 18 7
155 191 3
180 100 2
141 143 2
34 111 6
18 59 7
97 140 2
109 120 9
98 187 7
127 105 9
161 119 3
117 1 9
170 117 8
6 11 6
10 176 8
174 69 6
57 83 2
179 84 1
40 29 5
199 23 8
125 67 1
101 23 7
131 198 6
162 36 8
93 169 5
17 120 10
19 107 1
