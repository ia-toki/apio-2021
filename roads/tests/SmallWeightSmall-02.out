064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
1070 534 210 66 15 4 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
